Podcast/SeriesDetailViewController.swift

```diff
@@ -38,17 +38,11 @@ class SeriesDetailViewController: ViewController, SeriesDetailHeaderViewDelegate
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        seriesHeaderView = SeriesDetailHeaderView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: seriesHeaderViewMinHeight))
+        seriesHeaderView = SeriesDetailHeaderView(frame: CGRect(x: 0.0, y: 0.0, width: view.frame.height, height: seriesHeaderViewMinHeight))
```

Comment:
why would we want the header view width to be view.frame.height?

Reply:
oops